Mathlib/Data/Int/GCD.lean

```diff
@@ -96,14 +96,17 @@ theorem gcdB_zero_left {s : ℕ} : gcdB 0 s = 1 := by
 theorem gcdA_zero_right {s : ℕ} (h : s ≠ 0) : gcdA s 0 = 1 := by
   unfold gcdA xgcd
   obtain ⟨s, rfl⟩ := Nat.exists_eq_succ_of_ne_zero h
-  simp [xgcdAux_succ]
-#align nat.gcd_a_zero_right Nat.gcdA_zero_right
+  -- Porting note: `simp [xgcdAux_succ]` crashes Lean here
```

Comment:
That sounds like something that needs a lean issue. Did you file one, @ChrisHughes24 ?

Reply:
I haven't. I've no idea what's happening and it's difficult to make an MWE